.github/workflows/ci.yml

```diff
@@ -75,3 +75,39 @@ jobs:
           path: |
             target/scoverage-report/**/*
             target/test-reports/**/*
+
+  docker:
+    runs-on: ubuntu-latest
+    needs: sbt-build-test
+    if: ${{ github.ref == 'refs/heads/pg-store' }}
```

Comment:
Why not put this branch requirement on top? 
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

Since this action is all pg-store only.

Reply:
The `sbt-build-test` job should run on any branch and PR, building and pushing a docker image should only go on `pg-store` for now (`main` later, when this is merged)